sys/posix/sockets/posix_sockets.c

```diff
@@ -211,6 +211,8 @@ static int _ep_to_sockaddr(const struct _sock_tl_ep *ep,
 static int _sockaddr_to_ep(const struct sockaddr *address, socklen_t address_len,
                            struct _sock_tl_ep *out)
 {
+    assert(address != NULL);
```

Comment:
Then please change PR title and commit message, that this is a fix for cppcheck warnings.